pkg/hubble/parser/seven/http.go

```diff
@@ -66,7 +66,12 @@ func decodeHTTP(flowType accesslog.FlowType, http *accesslog.LogRecordHTTP, opts
 }
 
 func (p *Parser) httpSummary(flowType accesslog.FlowType, http *accesslog.LogRecordHTTP, flow *flowpb.Flow) string {
-	httpRequest := fmt.Sprintf("%s %s", http.Method, http.URL)
+	uri, _ := url.Parse(http.URL.String())
+	if p.opts.RedactHTTPQuery {
+		uri.RawQuery = ""
+		uri.Fragment = ""
+	}
+	httpRequest := fmt.Sprintf("%s %s", http.Method, uri)
```

Comment:
Let's avoid `fmt.Sprintf` in hot paths. This can be a simple string concat instead.

Ref: #19571

Reply:
Fixed: [4956bd1](https://github.com/cilium/cilium/pull/27553/commits/4956bd17e1b510bd8cb18e6bb43cdf218fbd5bb9)

Reply:
Thanks. Could you squash the commits together?

Reply:
Sure! Done :)